db/migrate/20230615184945_add_pinned_to_log_entries.rb

```diff
@@ -0,0 +1,5 @@
+class AddPinnedToLogEntries < ActiveRecord::Migration[6.1]
+  def change
+    add_column :log_entries, :pinned, :boolean, default: false
```

Comment:
```suggestion
    add_column :log_entries, :pinned, :boolean, default: false, null: false
```
It's good practice to disallow null values for boolean columns _unless_ it's meaningful. You will have to run `bin/rails db:migrate:redo` to update the schema as well.